OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java

```diff
@@ -67,9 +67,11 @@ public static File getSelectedFile(String extension, String description, DialogT
         int returnVal = 0;
         switch (type) {
             case SAVE:
+                chooser.setDialogTitle("Save As");
```

Comment:
Actually, I think I'm fine with leaving it as it is. It makes me wonder though, why we have separated SAVE and SAVEAS cases here. I don't think the behaviour between those two should ever be different. Can we merge those cases into one?